plugins/backstage-plugin-flux/README.md

```diff
@@ -277,6 +282,48 @@ kubernetes:
           caData: LS0tLS1CRUdJTiBDRVJUSUZJQ0...
 ```
 
+6. [Optional] Add a Flux Runtime page to your app
+
+- An example of a basic page can be found at [Flux Runtime Page](https://github.com/weaveworks/weaveworks-backstage/plugins/backstage-plugin-flux/src/components/FluxRuntimePage/index.tsx/).
```

Comment:
I'm not sure if this link will work, there is some /tree/ etc in there, we could skip it.

```suggestion
- An example Page is included as the `<FluxRuntimePage />` component. 
```